CMakeLists.txt

```diff
@@ -216,7 +216,11 @@ if(BUILD_IDLC)
 endif()
 
 option(BUILD_DOCS "Build documentation." OFF)
-add_subdirectory(docs)
+option(BUILD_SCHEMA "Build generated schema for configuration options." OFF)
+
+if(BUILD_DOCS)
+  add_subdirectory(docs)
```

Comment:
Shouldn't this be unconditional again?

Reply:
Answer to self: yes, otherwise it doesn't try to build it ... and when I try it fails on trang.

Ubuntu presumably has a package, homebrew has "jing-trang", chocolatey doesn't seem to have one. Arguably it isn't necessary to check this on each platform as it is platform independent.

But that little detail clearly needs some further investigation.

Reply:
Good catch